__tests__/responding_to_the_lack_of_relevant_messages_test.js

```diff
@@ -61,26 +61,22 @@ test('not responding on removed channels', () => {
     expect(sent).toBe(false);
 });
 
-test('responding with refusal when configuring an already configured channel', () => {
-    let message = '';
-    const bot = new Bot(120);
-    bot.configure('test-channel', function (msg) {
-        message = msg;
-    });
-    message = '';
+// test('responding with refusal when configuring an already configured channel', () => {
+//     let message = '';
+//     const bot = new Bot(120);
+//     bot.configure('test-channel', function (msg) {
+//         message = msg;
+//     });
+//     message = '';
 
-    bot.configure('test-channel', function () {});
+//     bot.configure('test-channel', function () {});
 
-    expect(message).toBe('Already autobumping!');
-});
+//     expect(message).toBe('Already autobumping!');
+// });
```

Comment:
Why is this one commented out?

Reply:
I pulled the start/stop messaging functionality out of the Bot class into index.js, and made the bot accept arbitrary argument-free callbacks when configured, so that test is no longer relevant. Wanted to see if you agree with that direction before writing a new functional test

Reply:
`test('overridding configuration when configuring an already configured channel', () => {
    let message = '';
    const bot = new Bot(120);

    bot.configure('test-channel', function () {
        message = 'should never happen';
    });

    bot.configure('test-channel', function () {
        message = 'verified'
    });

    bot.notify('test-channel', new Date(10000000));
    bot.reviveChannels(new Date(20000000));

    expect(message).toBe('verified');
});`